pravega/schema-registry
# Issue 121: Supplying init containers within helm charts

Signed-off-by: SrishT <Srishti.Thakkar@dell.com>

**Change log description**  
Adds support for `initContainers` and `prestartCommand` within the schema registry helm charts so that users can inject their auth handler implementation and perform any preconfiguration steps before deploying the schema registry pod.

**Purpose of the change**  
Fixes #121 

**What the code does**  
Adds the init container configuration provided by the user in the `values.yaml`. Also allows the user to provide the command that gets executed as part of the schema registry's entrypoint.sh script before actually starting the main process.

**How to verify it**  
Provide an init container configuration and a pre-start command for the main container. 
The init container should be able to run successfully before the main container starts execution, and the provided pre-start command should get executed as part of the entrypoint before the main process starts.
